docs/sdk-and-tools/indices/rating.md

```diff
@@ -0,0 +1,29 @@
+---
+id: rating
+title: rating
+---
+
+The `_id` field for this index is composed in this way: `{validator_bls_key}_{epoch}` (example: `blskey_37`).
+
+
+| Field     | Description                                                     |
+|-----------|-----------------------------------------------------------------|
+| rating    | The rating of a validator                                       |
```

Comment:
please add more details, such as the range. Example:
`The rating of the validator, wich can be in the [0, 100] range`

Reply:
done